tests/containers/accountLanding/AccountLandingContainer-test.jsx

```diff
@@ -0,0 +1,117 @@
+
+import * as FiscalYearHelper from 'helpers/fiscalYearHelper';
+import AccountLandingContainer from 'containers/accountLanding/AccountLandingContainer';
+
+import { mockData, mockParsed } from './mockFederalAccounts';
+
+jest.mock('helpers/accountLandingHelper', () => require('./accountLandingHelper'));
+
+// mock the child component by replacing it with a function that returns a null element
+jest.mock('components/accountLanding/AccountLandingContent', () =>
+    jest.fn(() => null));
+
+describe('AccountLandingContainer', () => {
+    it('should make an API request on mount', async () => {
+        // mount the container
+        const container = mount(<AccountLandingContainer />);
+        const parseAccounts = jest.fn();
+        container.instance().parseAccounts = parseAccounts;
+
+        await container.instance().accountsRequest.promise;
+
+        expect(parseAccounts).toHaveBeenCalledTimes(1);
+    });
+
+    describe('showColumns', () => {
+        it('should build the table', async () => {
+            // mount the container
+            const container = mount(<AccountLandingContainer />);
+
+            await container.instance().accountsRequest.promise;
+
+            // validate the state contains the correctly parsed values
+            const fy = FiscalYearHelper.defaultFiscalYear();
+            const expectedState = [
+                {
+                    columnName: "account_number",
+                    defaultDirection: "desc",
+                    displayName: "Account Number"
+                },
+                {
+                    columnName: "account_name",
+                    defaultDirection: "asc",
+                    displayName: "Account Name"
+                },
+                {
+                    columnName: "managing_agency",
+                    defaultDirection: "asc",
+                    displayName: "Managing Agency"
+                },
+                {
+                    columnName: "budgetary_resources",
+                    defaultDirection: "desc",
+                    displayName: `${fy} Budgetary Resources`
+                }
+            ];
+
+            expect(container.state().columns).toEqual(expectedState);
+        });
+    });
+
+    describe('parseAccounts', () => {
+        it('should parse the API response and update the container state', () => {
+            const container = mount(<AccountLandingContainer />);
+
+            container.instance().parseAccounts(mockData.data);
+            expect(container.state().results).toEqual(mockParsed);
+        });
+    });
+
+    describe('updateSort', () => {
+        it('should update the state and make an API request', async () => {
+            // mount the container
+            const container = mount(<AccountLandingContainer />);
+            const parseAccounts = jest.fn();
+            container.instance().parseAccounts = parseAccounts;
+
+            await container.instance().accountsRequest.promise;
```

Comment:
In this and the next test, I don't believe we should be making API calls and parsing responses. These tests should be narrowly focused: given an explicitly provided input to the method being tested, what is the expected behavior/output?

For things like pagination, we can explicitly set the container's state/props via Enzyme's `setState` or `setProps` (http://airbnb.io/enzyme/docs/api/ShallowWrapper/setState.html), call the method being tested with a specific input, and then validate either the method's return value or the container's new state/props (http://airbnb.io/enzyme/docs/api/ShallowWrapper/state.html).

This ensures that our tests are specific (so when they fail, we know the exact behavior that is failing rather than "something is broken in the container"). The other behavior, like calling the API or parsing its response, should be covered by the tests for those methods and *this* test will rely on those tests to assume they are functioning correctly.